server/controllers/list.js

```diff
@@ -1118,6 +1119,7 @@ const changeType = (req, resp) => {
   const { id: listId } = req.params;
   const { _id: currentUserId } = req.user;
   let cohortMembers;
+  let removedViewers;
```

Comment:
line 1123 should be together with const from line 1121

const
const
const 
let
let
let